models/cfp.py

```diff
@@ -770,7 +770,7 @@ def end_date(self) -> Optional[datetime]:
 
     @property
     def potential_end_date(self) -> Optional[datetime]:
-        start = self.start_date
+        start = self.potential_start_date
```

Comment:
Vaguely concerned about side effects here given this has clearly been load-bearing, but I can't see anything obvious that was depending on the incorrect behaviour. Seems to only touch the clashfinder and attendee content.

Reply:
Actually overlaps_with is just going to be wrong if one thing has a potential time and the other doesn't. I need to think about what the correct behaviour should be there.

Reply:
Ah, context here from git digging: This was written before we had potential times, and at that point _everything_ had a potential time or it didn't.

It's still wrong but it doesn't really matter. This is fine.